pubspec.yaml

```diff
@@ -68,3 +69,7 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.io/custom-fonts/#from-packages
+  fonts:
+    - family:  CustomIcons
+      fonts:
+        - asset: fonts/CustomIcons.ttf
```

Comment:
I think the fonts should go in the asset folder along with all other assets. Better organization that way. meaning the path for this would be like `/assets/fonts/CustomIcons.ttf`